.obs/chartfile/operator/values.yaml

```diff
@@ -9,10 +9,15 @@ seedImage:
   tag: "%VERSION%"
   imagePullPolicy: IfNotPresent
 
-channel:
-  name: "sle-micro-5.5"
-  image: "%%IMG_REPO%%/rancher/elemental-channel/sle-micro"
-  tag: "5.5"
+# default Elemental channels
+defaultChannels:
+  # default channels are included in the chart
+  included: true
+  # default channels to include
+  includes:
+    sle_micro_5_5: true
```

Comment:
I like the idea of making it more generic, but I'm struggling to make it working nice with the questions.
I also think we don't need to make the default channels configurable, we can hardcode all the values, as it's an opinionated choice. 

I'll fix the snake_case variables as you mentioned and see if it can be more generic.

Reply:
![Screenshot from 2024-07-29 13-08-29](https://github.com/user-attachments/assets/ac680a57-469a-4ab8-9679-91932c700d22)

Reply:
Picture above is how it looks like after the changes.
I made the structure a bit more generic. 

I'm still playing out with the `show_if`. I wonder if it's even necessary. Since chart and questions are linked, the expected map keys should always be found, `show_if` only plays nice if we somehow keep the question but remove the default values, which I hope we won't. :D

I also exposed a boolean flag only to include/exclude the channels, further settings can be edited through values, but not from the Rancher UI when installed. I think this is reasonable, otherwise we will have too many options and checkboxes when we assume most users will use the default values.

Reply:
Just out of curiosity I tested the following and it does not show in the UI as expected:

```yaml
- variable: defaultChannels.doesNotExist.included
  show_if: "defaultChannels.doesNotExist"
  default: true
  description: "Foo bar"
  type: boolean
  label: Just for testing
  group: "Default Elemental OS Channels"
```
So I think we can keep the `show_if` just for safety.